modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -583,7 +583,7 @@ export default {
     transform: translate(-50%, 0);
   }
 
-  .apos-area-widget-inner .apos-area-widget-inner /deep/ .apos-context-menu__btn {
+  .apos-area-widget-inner .apos-area-widget-inner /deep/ .apos-context-menu__btn .apos-button {
```

Comment:
That's a deep boi. Doesn't look like there's an easy alternative. Unless we pass a class into the button?

Reply:
does the Vue component receive any of the 'normal' element attributes that get tacked onto the `<Component>`? .. if so, we could generate a child class based on the one first attached, that might shortcut some of this

Reply:
There are probably ways, but it'd be some decent refactoring. Not a fight for this day.